src/NzbDrone.Core/ImportLists/Simkl/SimklParser.cs

```diff
@@ -43,7 +43,7 @@ public virtual IList<ImportListItemInfo> ParseResponse(ImportListResponse import
                 {
                     var tentativeTvdbId = int.TryParse(show.Show.Ids.Tvdb, out var tvdbId) ? tvdbId : 0;
 
-                    if (tentativeTvdbId > 0 && show.AnimeType == SimklAnimeType.Tv)
+                    if (tentativeTvdbId > 0 && show.AnimeType != SimklAnimeType.Movie && show.AnimeType != SimklAnimeType.Music)
```

Comment:
Leaving this to compare to Tv makes it easier to read and doesn't need to be updated every time a new value is added.

Reply:
We were accounting for the different anime types ONAs, OVAs, and Specials if they had a TVDB id. Should we not include them? Cyberpunk: Edgerunners is listed as an ONA, for example.

Reply:
In that case opting into the ones we want is probably safer than excluding the ones we don't

Reply:
That makes sense. I will do that.